src/main/java/mpicbg/spim/postprocessing/deconvolution2/LRFFT.java

```diff
@@ -390,8 +390,8 @@ public void setKernel( final Image<FloatType> kernel )
 	public Image<FloatType> getKernel1() { return kernel1; }
 	public Image<FloatType> getKernel2() { return kernel2; }
 	
-	public void setCurrentIteration( final int i ) { this.i = i; }
-	public int getCurrentIteration() { return i; }
+	public void setCurrentIteration( final int i ) { this.iteration = i; }
```

Comment:
The `this.` prefix now becomes unnecessary. BTW I really like the more descriptive name `iteration`.